src/modules/differential_drive/DifferentialDriveGuidance/DifferentialDriveGuidance.cpp

```diff
@@ -117,19 +116,6 @@ void DifferentialDriveGuidance::computeGuidance(float yaw, float angular_velocit
 	output.closed_loop_speed_control = output.closed_loop_yaw_rate_control = true;
 	output.timestamp = hrt_absolute_time();
 
```

Comment:
Could it be if the the heading is already aligned that the trajectory is already going ahead but the spoolup stops the vehicle from moving? That could lead to a "jump" once the spoolup is done 🤔

Reply:
Could also happen when disarmed or when turning (https://github.com/PX4/PX4-Autopilot/pull/22550#discussion_r1486079443), similar to integrator windup.

Reply:
I think we can follow up with corner case improvements in another pr.